### PR TITLE
Chop longer material names in smaller pieces in print-job-info.

### PR DESCRIPTION
**Pull request by rburema**
[CURA-6141]

### CODE REVIEW

File: resources/qml/ActionPanel/PrintJobInformation.qml [hunk 1]
```diff
@@ -90,14 +90,31 @@ Column
             property var printMaterialCosts: PrintInformation.materialCosts
             property var printMaterialNames: PrintInformation.materialNames
 
+            function putBreaksIfLong(item)
+            {
+                var result = ""
+
+                const max_length = 13  // No longer than 13 -> 'WWWWWWWWWWWWWW' doesn't fit.
```
Comment (fieldOfView) [inline resources/qml/ActionPanel/PrintJobInformation.qml]: Would it be a good idea to better document the magic number, perhaps make it a constant of sorts defined at the top of the file?

**Comment by nallath** [comment]
I guess an even better way is to do away with the HTML altogether and put it in actual QML. I don't really see a reason why it's in HTML table code in the first place.

File: resources/qml/ActionPanel/PrintJobInformation.qml (hunk 1 above)
Comment (rburema) [reply resources/qml/ActionPanel/PrintJobInformation.qml]: Probably, if we decide that it should remain HTML (see below).

**Comment by rburema** [comment]
I've rewritten this to use QML-elements instead of HTML-tables, see #5224

**Pull request closed without merging**